hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -275,16 +275,29 @@
                                         </xs:documentation>
                                     </xs:annotation>
                                 </xs:attribute>
-                                <xs:attribute name="optimize-queries" use="optional" type="parameterized-boolean"
-                                              default="false">
+                                <xs:attribute name="optimize-queries" use="optional" type="parameterized-boolean">
                                     <xs:annotation>
                                         <xs:documentation>
+                                            This parameter is deprecated as of Hazelcast 3.6
+                                            Use cache-deserialized-values attribute instead.
+
                                             This parameter is used to increase the speed of query processes in the map.
                                             It only works when `in-memory-format` is set as `BINARY` and performs
                                             a pre-caching on the entries queried.
                                         </xs:documentation>
                                     </xs:annotation>
                                 </xs:attribute>
+                                <xs:attribute name="cache-deserialized-values" use="optional" type="cache-deserialized">
+                                    <xs:annotation>
+                                        <xs:documentation>
+                                            Control caching of de-serialized values. Caching makes query evaluation faster, but it cost memory.
+                                            Possible Values:
+                                            NEVER: Never cache de-serialized object
+                                            INDEX-ONLY: Caches values only when they are inserted into an index.
```

Comment:
Small inconsistenty: NEVER and ALWAYS descriptions are written in imperative thus it would be better to use it in all descrpitions like: `Cache values...` without `s`